TWLight/applications/forms.py

```diff
@@ -209,27 +209,25 @@ def _add_user_data_subform(self, user_data):
                 # Show which partner wants which personal data if applying
                 # for more than one.
                 if len(self.field_params) > 1:
+                    # fmt: off
                     # Translators: This text is shown in the application form under each piece of personal information requested. {partner_list} will be a list of 2 or more organisations that require this personal data, and should not be translated.
-                    self.fields[datum].help_text = _(
-                        "Requested by: {partner_list}".format(
+                    self.fields[datum].help_text = _("Requested by: {partner_list}".format(
                             partner_list=", ".join(user_data[datum])
                         )
                     )
+                    # fmt: on
                 user_data_layout.append(datum)
 
             self.helper.layout.append(user_data_layout)
-            # Translators: This this note appears in a section of a form where we ask users to enter info (like country of residence) when applying for resource access.
-            self.helper.layout.append(
-                HTML(
-                    _(
-                        "<p><small><i>Your personal data "
-                        'will be processed according to our <a href="{terms_url}">'
-                        "privacy policy</a>.</i></small></p>".format(
-                            terms_url=reverse("terms")
-                        )
-                    )
-                )
+            # fmt: off
+            # Translators: This note appears in a section of a form where we ask users to enter info (like country of residence) when applying for resource access.
+            disclaimer_html = _("<p><small><i>"
+                "Your personal data will be processed "
+                "according to our <a href='{terms_url}'> privacy "
+                "policy</a>.</i></small></p>".format(terms_url=reverse("terms"))
             )
```

Comment:
closing brackets for closing brackets for `_()` should be lifted to the same line as the source string`_()` should be lifted to the same line as the source string